commands/create.go

```diff
@@ -0,0 +1,152 @@
+package commands
+
+import (
+	"bufio"
+	"fmt"
+	"github.com/chermehdi/egor/config"
+	"github.com/fatih/color"
+	"github.com/urfave/cli/v2"
+	"io"
+	"os"
+	"strings"
+)
+
+func ReadLine(reader io.Reader) string {
+	result := ""
+	r := bufio.NewReader(reader)
+	for true {
+		readBytes, isPrefix, _ := r.ReadLine()
+		result += string(readBytes)
+		if !isPrefix {
+			break
+		}
+	}
+	return result
+}
+
+func askInt(reader io.Reader, question string, defaultValue float64) float64 {
+	color.Green("%s (%d by default): ", question, int(defaultValue))
+	input := ReadLine(reader)
+	var value = -1
+	n, err := fmt.Sscanf(input, "%d", &value)
+	if err != nil || n == 0 || value == -1 {
+		return defaultValue
+	}
+	return float64(value)
+}
+
+func askBool(reader io.Reader, question string, defaultValue bool) bool {
+	value := "N"
+	if defaultValue {
+		value = "Y"
+	}
+	response := ask(reader, question, value)
+	return response == "Y" || response == "y"
+}
+
+func ask(reader io.Reader, question, defaultValue string) string {
+	color.Green("%s (%s by default): ", question, defaultValue)
+	value := ReadLine(reader)
+	if value == "" {
+		return defaultValue
+	}
+	return value
+}
+
+func askEndOfFile(question string) string {
+	color.Green(question)
+	scanner := bufio.NewScanner(os.Stdin)
+	result := ""
+	for scanner.Scan() {
+		result += scanner.Text()
+		result += "\n"
+	}
+	return result
+}
+
+// Fill the given task reference according to input coming from the reader object.
+func fillTaskFromQuestions(task *config.Task, reader io.Reader) error {
+	task.Name = ask(reader, "Task Name?", task.Name)
+	task.TimeLimit = askInt(reader, "Time limit in ms?", task.TimeLimit)
+	testCases := askInt(reader, "Number of test cases?", 0)
+	for i := 0; i < int(testCases); i++ {
+		input := askEndOfFile(fmt.Sprintf("Enter testcase #%d input:\n", i))
+		output := askEndOfFile(fmt.Sprintf("Enter testcase #%d expected output:\n", i))
+		task.Tests = append(task.Tests, config.TestCase{
+			Input:  input,
+			Output: output,
+		})
+	}
+	// TODO(chermehdi): This information is irrelevant now, because we don't do anything special with the languages
```

Comment:
If not used then better remove it. Its confusing if the user doen't know what is it for.

Reply:
Well the problem here, is that it should be used, and it will be in some future feature.